plugins/callback/profile_roles.py

```diff
@@ -116,3 +116,4 @@ def playbook_on_stats(self, stats):
         msg_total = u"{0:-<70}{1:->9}".format(u'total ', u' {0:.02f}s'.format(total_time))
         self._display.display(filled("", fchar="~"))
         self._display.display(msg_total)
+        
```

Comment:
To pass the sanity test, you need to remove the whitespace at the end of the line.